docker/cpu/Dockerfile

```diff
@@ -23,8 +23,14 @@ ENV PATH=/opt/conda/bin:$PATH
 
 RUN conda install -y -c conda-forge pyqt uv && \
```

Comment:
why didn't you use
`conda install -y -c conda-forge -c gurobi pyqt uv gurobi`
?
Your solution looks very hacky

Reply:
@JoOkuma it is hacky, but the problem is that gurobi and gurobipy arent identified as the same package. So when I install ultrack, it doesn't find gurobipy and installs it from pypi. For that reason I need to remove it after is installed and install gurobi from conda.

If I install gurobi earlier, when I do `pip uninstall gurobipy` it will remove the conda package :melting_face:

Reply:
that sucks, well, you found a solution